crates/jstzd/tests/jstzd_test.rs

```diff
@@ -27,11 +27,18 @@ async fn jstzd_test() {
 
     fetch_config_test(config, jstzd_port).await;
 
-    reqwest::Client::new()
-        .put(&format!("http://localhost:{}/shutdown", jstzd_port))
-        .send()
+    tokio::spawn(async move {
+        tokio::time::sleep(tokio::time::Duration::from_secs(10)).await;
+        reqwest::Client::new()
+            .put(&format!("http://localhost:{}/shutdown", jstzd_port))
+            .send()
+            .await
+            .unwrap();
+    });
+
+    tokio::time::timeout(tokio::time::Duration::from_secs(30), jstzd.wait())
         .await
-        .unwrap();
+        .expect("should not wait too long for the server to be taken down");
```

Comment:
maybe we can create a separate test for a shut down ?

Reply:
why?

Reply:
i just realized jstzd server is only shut down via the shutdonw signal and otherwise will keep running as a daemon so it will never `Drop`